### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by dacook**
Let me know what you think

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -186,36 +190,43 @@ Switch application:
...
-- [ ] make new mysql writable
-- [ ] change database.yml
+- [ ] check databases are in sync (eg check your user updated_at after visiting a page)
+- [ ] make new mysql db writable
+- [ ] change new database.yml to point to local db server
 - [ ] stop delayed job
+- [ ] confirm no other scheduled tasks currently running or about to run
+- [ ] Set old master to read-only
 - [ ] reload new application
 - [ ] make new mysql master `STOP SLAVE; RESET MASTER;`
```
Comment (mkllnk) [inline README.md:211]: Depending on the research above, we may need to deactivate any persisted master-slave config for restarts.

File: README.md [hunk 2]
```diff
@@ -186,36 +190,43 @@ Switch application:
...
 - [ ] start delayed job
 
 Finishing it off:
-- [ ] update post-receive hook on new server
-- [ ] install monit on new server
+- [ ] install monit on new server (copy config from old: /etc/monit/conf.d/rails-fairfood)
 - [ ] change DNS entry
 - [ ] copy TLS certificates
+- [ ] update post-receive hook on new server:
```
Comment (mkllnk) [inline README.md:218]: The snippet below is for the post-receive hook on the old server.

Comment (dacook) [reply README.md]: Good spotting! Fixed.

**Comment by dacook** [comment]
Hey @mkllnk , I fixed the checklist, happy with the other changes too?

**Review by mkllnk — APPROVED** [review]
Great!

**Pull request merged**